homeassistant/components/light/mqtt.py

```diff
@@ -223,6 +223,8 @@ def __init__(self, name, unique_id, effect_list, topic, templates,
         self._supported_features = 0
         self._supported_features |= (
             topic[CONF_RGB_COMMAND_TOPIC] is not None and SUPPORT_COLOR)
+        self._supported_features |= (
+            topic[CONF_RGB_COMMAND_TOPIC] is not None and SUPPORT_BRIGHTNESS)
```

Comment:
**Preference:**
Let's just roll this into the line above: `and (SUPPORT_COLOR | SUPPORT_BRIGHTNESS)`.